emukit/core/loop/model_updaters.py

```diff
@@ -24,19 +26,27 @@ def update(self, loop_state: LoopState) -> None:
 
 class FixedIntervalUpdater(ModelUpdater):
     """ Updates hyper-parameters every nth iteration, where n is defined by the user """
-    def __init__(self, model: IModel, interval: int = 1) -> None:
+    def __init__(self, model: IModel, interval: int=1, targets_extractor_fcn: Callable=None) -> None:
         """
         :param model: Emukit emulator model
         :param interval: Number of function evaluations between optimizing model hyper-parameters
+        :param targets_extractor_fcn: A lambda function that takes in loop state and returns the training targets.
```

Comment:
Does not have to be lambda, right? Any function would do
```suggestion
        :param targets_extractor_fcn: A function that takes in loop state and returns the training targets.
```